protocol/0009-MRKP-mark_price.md

```diff
@@ -2,16 +2,20 @@ Feature name: Mark Price
 # Summary
 A *Mark Price* is a concept derived from traditional markets.  It is a calculated value for the 'current market price' on a market.
 
+Introduce a network parameter `network.markPriceUpdateMaximumFrequency` with minimum allowable value of `0s` maximum allowable value of `1h` and a default of `5s`. 
```

Comment:
Isn't it just `network.markPriceUpdateFrequency`? Or are you trying to convey that exact timings are not guaranteed?